eloquent/ci-nightly-navigation2-prerequisites.yaml

```diff
@@ -0,0 +1,106 @@
+%YAML 1.1
+# ROS buildfarm ci-build file
+---
+build_environment_variables:
+  CMAKE_PREFIX_PATH: '/opt/ros/melodic:$CMAKE_PREFIX_PATH'
+  LD_LIBRARY_PATH: '/opt/ros/melodic/lib:$LD_LIBRARY_PATH'
+  NDDSHOME: '/opt/rti.com/rti_connext_dds-5.3.1'
```

Comment:
This isn't needed for a non-Connext build.

Same for `RTI_NC_LICENSE_ACCEPTED` below and for the other config file.

Reply:
I think it is more important to encourage a Dockerfile cache hit with the `nightly-fastrtps` job, though what you're saying indicates that that job should be changed as well.

Reply:
The `nightly-fastrtps` job doesn't have these env vars since it doesn't need them: https://github.com/ros2/ros_buildfarm_config/blob/0587e6b59d3a427ac1731c18c629eb0f34402a97/eloquent/ci-nightly-fastrtps.yaml#L5-L9

Reply:
My branch is stale, apologies. I'll align with the updated version of `ci-nightly-fastrtps.yaml`.

Reply:
Fixed.